features/step_definitions/edit_profile_steps.rb

```diff
@@ -42,6 +42,10 @@
   click_button button
 end
 
+When(/^They upload the image$/) do
+  attach_file('profile[avatars][]', 'public/test_image/pokemon.jpg')
+end
+
 Then(/^They see the message "([^"]*)"$/) do |message|
   page.should have_selector ".alert", text: message
```

Comment:
This is fine but I don't think it actually verifies either that images were saved or are displayed in the profile. I've found some issues with strong parameters, I'll look into it and see if I can figure out what's wrong